cluster/examples/kubernetes/ceph/upgrade-from-v1.4-crds.yaml

```diff
@@ -1315,3 +1315,109 @@ spec:
       subresources:
         status: {}
 # OLM: END OBJECTBUCKETCLAIM CRD
+---
```

Comment:
With #6623 we no longer have this upgrade crds manifest, it's all just crds.yaml